e2e/app/page/workspace-edit-page.ts

```diff
@@ -424,11 +425,20 @@ export default class WorkspaceEditPage extends AuthenticatedPage {
   /**
    * Find and click the CREATE WORKSPACE (FINISH) button
    */
-  async clickCreateFinishButton(): Promise<void> {
-    const createButton = await this.getCreateWorkspaceButton();
-    await createButton.focus(); // bring into viewport
-    await this.clickAndWait(createButton);
+  async clickCreateFinishButton(button: ElementHandle | Button): Promise<string> {
+    await button.focus(); // bring into viewport
+    await button.click();
+
+    // confirm create in pop-up dialog
+    const dialog = new Dialog(this.page);
+    const dialogText = await dialog.getContent();
+    await Promise.all([
+      dialog.clickButton(ButtonLabel.Confirm),
+      dialog.waitUntilDialogIsClosed(),
+      this.page.waitForNavigation({waitUntil: ['domcontentloaded', 'networkidle0'], timeout: 60000}),
+    ]);
     await this.waitUntilNoSpinner();
+    return dialogText;
   }
```

Comment:
This method used to work only for the `CREATE WORKSPACE` button on new workspace form. There is a separate methods for dealing with `DUPLICATE WORKSPACE` button on duplicate workspace form. To avoid boilerplate code, it is now just one method that handles both button, waiting and pop-up dialog.